pkg/apis/pg.percona.com/v2beta1/perconapgcluster_types.go

```diff
@@ -131,7 +135,17 @@ type PerconaPGClusterSpec struct {
 	PMM *PMMSpec `json:"pmm,omitempty"`
 }
 
+type AppState string
+
+const (
+	AppStateInit  AppState = "initializing"
+	AppStateReady AppState = "ready"
+	AppStateError AppState = "error"
+)
+
 type PerconaPGClusterStatus struct {
+	State                                AppState `json:"state"`
+	Host                                 string   `json:"host"`
```

Comment:
Regarding having a column to tell us running PG instances (like we have in other operators), here we need to add something like `PGReadyInstaces` , or just `PG` or maybe a struct that would hold `size` and `ready` replicas. But this feels like we duplicate things in our status since we have this info from Crunchy (inside `PostgresClusterStatus.InstanceSets`). But unfortunately, we cant use this info directly to populate the column since we need to go through each `instanceSet` and calculate all of the ready pods, kubebuilder does not support something like that.

Need your input here, I don't like to duplicate this data and bloat our status, but I also don't like that our columns would not be consistent between the operators.

Reply:
I discussed it with Diogo and we can leave it out for now and fix later. 

`host` and `state` are critical.

Reply:
Also @inelpandzic , should it be `STATUS` in the short output? As I see it is `STATUS` in psmdb:
```
NAME              ENDPOINT                                        STATUS   AGE
my-cluster-name   my-cluster-name-rs0.default.svc.cluster.local   ready    4h27m
```

Reply:
Yes @spron-in , it is `STATUS`, my example in the description was wrong.

Reply:
@inelpandzic it may be good idea to remove Crunchy's status from ours, I don't think there's a need to mirror their CR's status to ours.

Reply:
Well now that you mentioned it, I do think it is a good idea, but I wouldn't do it in this task, as it may require more careful consideration. If you see we mirror a lot of them in our CR that goes well with the status.